manifests/fluentd/fluentd-cluster-cfg-output-buffer-example.yaml

```diff
@@ -63,7 +63,7 @@ spec:
       type: file
       path: /buffers/stdout.log
   - elasticsearch:
-      host: elasticsearch-logging-data.kubesphere-logging-system.svc
+      host: elasticsearch-logging-data.fluent.svc
```

Comment:
we can keep the es namespace as it is before : elasticsearch-logging-data.kubesphere-logging-system.svc